kntest/pkg/cluster/options.go

```diff
@@ -0,0 +1,41 @@
+/*
+Copyright 2020 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"github.com/spf13/cobra"
+
+	"knative.dev/test-infra/kntest/pkg/cluster/ops"
+)
+
+func addOptions(clusterCmd *cobra.Command, rw *ops.RequestWrapper) {
+	pf := clusterCmd.PersistentFlags()
+	req := &rw.Request
+	pf.Int64Var(&req.MinNodes, "min-nodes", 0, "minimal number of nodes")
```

Comment:
min & max nodes should have good defaults

Reply:
Add that as a comment?

Reply:
Added the comment